geomesa-quickstart-hbase/src/main/java/com/example/geomesa/hbase/HBaseQuickStart.java

```diff
@@ -0,0 +1,249 @@
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+public class HBaseQuickStart {
+    static String TABLE_NAME = "bigtable.table.name".replace(".", "_");
+
+    // sub-set of parameters that are used to create the HBase DataStore
+    static String[] HBASE_CONNECTION_PARAMS = new String[]{
+            TABLE_NAME
+    };
+
+    /**
+     * Creates a common set of command-line options for the parser.  Each option
+     * is described separately.
+     */
+    static Options getCommonRequiredOptions() {
+        Options options = new Options();
+
+        Option tableNameOpt = OptionBuilder.withArgName(TABLE_NAME)
+                .hasArg()
+                .isRequired()
+                .withDescription("table name")
+                .create(TABLE_NAME);
+        options.addOption(tableNameOpt);
+
+        return options;
+    }
+
+    static Map<String, Serializable> getHBaseDataStoreConf(CommandLine cmd) {
+        Map<String , Serializable> dsConf = new HashMap<>();
+        for (String param : HBASE_CONNECTION_PARAMS) {
+            dsConf.put(param.replace("_", "."), cmd.getOptionValue(param));
+        }
+        return dsConf;
+    }
+
+    static SimpleFeatureType createSimpleFeatureType(String simpleFeatureTypeName)
+            throws SchemaException {
+
+        // list the attributes that constitute the feature type
+        List<String> attributes = Lists.newArrayList(
+                "Who:String",
+                "What:java.lang.Long",     // some types require full qualification (see DataUtilities docs)
+                "When:Date",               // a date-time field is optional, but can be indexed
+                "*Where:Point:srid=4326",  // the "*" denotes the default geometry (used for indexing)
+                "Why:String"               // you may have as many other attributes as you like...
+        );
+
+        // create the bare simple-feature type
+        String simpleFeatureTypeSchema = Joiner.on(",").join(attributes);
+        SimpleFeatureType simpleFeatureType =
+                DataUtilities.createType(simpleFeatureTypeName, simpleFeatureTypeSchema);
+
+        return simpleFeatureType;
+    }
+
+    static FeatureCollection createNewFeatures(SimpleFeatureType simpleFeatureType, int numNewFeatures) {
+        DefaultFeatureCollection featureCollection = new DefaultFeatureCollection();
+
+        String id;
+        Object[] NO_VALUES = {};
+        String[] PEOPLE_NAMES = {"Addams", "Bierce", "Clemens"};
+        Long SECONDS_PER_YEAR = 365L * 24L * 60L * 60L;
+        Random random = new Random(5771);
+        DateTime MIN_DATE = new DateTime(2014, 1, 1, 0, 0, 0, DateTimeZone.forID("UTC"));
+        Double MIN_X = -78.0;
+        Double MIN_Y = -39.0;
```

Comment:
[guilty admission]  This is only in the Southern Hemisphere, because I accidentally included the negative sign in the original GeoMesa quick-start.  Any chance we could fix this (as intended), and remove the sign?  Not a big deal.